src/js/models/v2/awardsV2/BaseCFDA.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * BaseCFDA.js
+ * Created by Jonathan Hill 03/17/20
+ */
+
+import { truncate, isNumber } from 'lodash';
+import { calculateTreemapPercentage, formatMoney } from 'helpers/moneyFormatter';
+
+const BaseCFDA = {
+    populate(data) {
+        this.samWebsite = data.sam_website || '';
+        this.cfdaWebsite = data.cfda_website || '';
+        this.cfdaFederalAgency = data.cfda_federal_agency || '';
+        this.cfdaNumber = data.cfda_number || '';
+        this.cfdaTitle = data.cfda_title || '--';
+        this.applicantEligibility = data.applicant_eligibility || '';
+        this.beneficiaryEligibility = data.beneficiary_eligibility || '';
+        this.cfdaObjectives = data.cfda_objectives || '';
+        this._totalFundingAmount = isNumber(data.total_funding_amount) ? data.total_funding_amount : '';
```

Comment:
It looks like `total_funding_amount` currently represents `federal_action_obligation_amount` + `non_federal_funding_amount` for the individual CFDA, which should not be used for the `percentOfTotal` calculation. We could either add another property to `cfda_info` with the total for all CFDAs associated with the given award, or calculate the total on the frontend.